static/src/js/actions/colorMaps.js

```diff
@@ -44,11 +43,6 @@ export const getColorMap = (payload) => async (dispatch, getState) => {
     })
     .catch((error) => {
       dispatch(setColorMapsErrored({ product }))
-      dispatch(actions.handleError({
-        error,
-        action: 'getColorMap',
-        resource: 'colormaps',
-        requestObject
-      }))
+      console.log(error)
```

Comment:
Does it still get reported to Splunk with the error handler removed?

Reply:
All the console logs get reported to splunk if they originate from a lambda. You'd just filter by the log_group they're also in the cloud-watch logs in fact we just forward those cloud-watch logs to Splunk because it has better filtering and searching